README.md

```diff
@@ -199,6 +200,26 @@ Examples:
 
 `server-ip` -> `TRENTO_SERVER_IP=https://localhost ./trento-agent start`
 
+## Commands
```

Comment:
Totally agree that there is the risk that it could get outdated very soon.
Indeed my intent is to just give a little reference, not to document everything, keeping the change rate of the doc at minimum :sweat_smile:  

I took inspiration from [datadog agent documentation](https://github.com/DataDog/documentation/blob/master/content/en/agent/guide/agent-commands.md) and looked reasonable. 
No problems in removing it, let's hear from the others as well.

Reply:
+1 removing this, we will forget about updating it for sure.

Reply:
gone :wink: